source/formcreator/targets.rst

```diff
@@ -129,6 +129,21 @@ The change is built from scratch. You may
 * set the urgency from a value or a question
 * choose the entity of the change among many policies
 * set the time to resolve
+
```

Comment:
Is this expected? I guess you'll end with a new list (ul) here

Reply:
no. fixed